packages/nimble-components/src/anchor-button/tests/anchor-button.stories.ts

```diff
@@ -0,0 +1,131 @@
+import type { Meta, StoryObj } from '@storybook/html';
+import { withXD } from 'storybook-addon-xd-designs';
+import { html, when } from '@microsoft/fast-element';
+import { createUserSelectedThemeStory } from '../../utilities/tests/storybook';
+import '../../all-components';
+import {
+    ButtonAppearance,
+    ButtonAppearanceVariant
+} from '../../patterns/button/types';
+
+interface AnchorButtonArgs {
+    label: string;
+    href: string;
+    appearance: string;
+    appearanceVariant: string;
+    icon: boolean;
+    endIcon: boolean;
+    contentHidden: boolean;
+    disabled: boolean;
+}
+
+const appearanceVariantDescription = `
+This attribute has no effect on anchor buttons with a \`ghost\` appearance.
+
+<details>
+    <summary>Primary Usage</summary>
+    Make a button primary to distinguish it visibly for one of the following reasons:
+    <ul>
+        <li>to indicate the action that allows the user to accomplish their most common or important goal</li>
+        <li>to indicate the action that allows the user to complete their task</li>
+    </ul>
+</details>`;
+
+const endIconDescription = `
+When including an icon after the text content, set \`slot="end"\` on the icon to ensure proper styling.
+
+This icon will be hidden when \`content-hidden\` is set.
+`;
+
+const metadata: Meta<AnchorButtonArgs> = {
```

Comment:
We should add a description of what the anchor button is used for (i.e. a component with the visual appearance of a button that navigates like a link when pressed)

Reply:
Done